start/sw/web-app/laravel5.md

```diff
@@ -40,6 +44,8 @@ We will need to set it up for easier development. Valet is original designed for
 For More Information, you can check them out here: 
 [Valet Linux](https://cpriego.github.io/valet-linux/)  and  [Laravel Valet](https://laravel.com/docs/5.7/valet)
 
+You can also setup Laravel using [Vagrant](http://developer.fedorainfracloud.org/tools/vagrant/about.html), [Docker](http://developer.fedorainfracloud.org/tools/docker/about.html).
```

Comment:
Please remove the website name ``http://developer.fedorainfracloud.org`` from the link.

Note that you're linking to the development instance!